Bottom: 6824ce2612271ccfc34e5dd0b2f6b3784194d87b
Top:    9b7286b347f015e4030f5608faefce9e62064dee
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2012-12-24 00:45:33 +0000

find.c: Pass flags to sub-functions without using static variable "TheFlag".


---

diff --git a/src/find.c b/src/find.c
index 3b551d4..e10cde2 100644
--- a/src/find.c
+++ b/src/find.c
@@ -268,7 +268,6 @@ typedef struct
  * some local identifiers
  */
 static Coord Bloat = 0;
-static int TheFlag = FOUNDFLAG;
 static void *thing_ptr1, *thing_ptr2, *thing_ptr3;
 static int thing_type;
 static bool User = false;    /* user action causing this */
